tests/api.js

```diff
@@ -1,22 +1,26 @@
-/**
- * Copyright (c) 2015 IBM Cloudant, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
- * except in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the
- * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied. See the License for the specific language governing permissions
- * and limitations under the License.
- */
+// Copyright © 2017 IBM Corp. All rights reserved.
```

Comment:
In fact looks like the eslint fix you ran might have replaced this all over the place which is bad.